lib/ios-deploy.js

```diff
@@ -64,25 +64,23 @@ class IOSDeploy {
     const afcService = await services.startAfcService(this.udid);
     try {
       const bundlePathOnPhone = await this.createAppPath(afcService, app);
+      const promises = [];
       await fs.walkDir(app, true, async (itemPath, isDir) => {
         const pathOnPhone = path.join(bundlePathOnPhone, path.relative(app, itemPath));
         if (isDir) {
           await afcService.createDirectory(pathOnPhone);
         } else {
-          const readStream = fs.createReadStream(itemPath, { autoClose: true });
-          const writeStream = await afcService.createWriteStream(pathOnPhone, { autoDestroy: true });
-          const filePush = new B((resolve, reject) => {
-            writeStream.on('close', resolve);
-            writeStream.on('error', reject);
-          });
+          const readStream = fs.createReadStream(itemPath, {autoClose: true});
+          const writeStream = await afcService.createWriteStream(pathOnPhone, {autoDestroy: true });
+          promises.push(new B((resolve) => writeStream.on('close', resolve)));
```

Comment:
In this version, what happens if `writeStream` errors?

Reply:
Good point. I will push that later on